_ml-commons-plugin/pretrained-models.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Pre-trained models
+parent: Model-serving framework
+nav_order: 120
+---
+
+# Pretrained models
+
+The model-serving framework supports a variety of open-source pretrained models that can assist with a variety of machine learning (ML) search and analytics use cases. 
+
+## Uploading pretrained models
+
+To use a pretrained model in your OpenSearch cluster:
+
+1. Select the model and download the model artifact and config.json file. For a list of pretrained models, see [supported pretrained model](#supported-pretrained-models).
+2. Upload the model using the [upload API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch). For more information on how to upload and use model, see [Model-serving framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework).
+
+## Supported pretrained models
+
+The model-serving framework supports the following models, categorized based on type. Although models with the same type will have similar use cases, each model has a different model size and performs differently depending on your cluster. For a comparison of the performances some pre-trained models, see the [sbert documentation](https://www.sbert.net/docs/pretrained_models.html#model-overview).
+
+
+### Sentence transformers
+
+Sentence transformer models map sentences and paragraphs across a dimensional dense vector space. The number of vectors depends on the model. Use these models for use cases such as clustering and semantic search.
+
+| **Model name** | **Vector dimensions** | **Torchscript artifact** | **ONNX artifact** |
```

Comment:
We should introduce the table in the text that precedes it. "The following table provides..."